src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -0,0 +1,30 @@
+import {addDays, addMonths, differenceInMonths, format, isValid} from 'date-fns';
+import CONST from '@src/CONST';
+
+/**
+ * Get the next billing date.
+ *
+ * @param initialDate - The initial billing date in 'yyyy-MM-dd' format.
+ * @returns - The next billing date in 'yyyy-MM-dd' format.
+ */
+function getNextBillingDate(initialDate: string): string {
+    let start = new Date(`${initialDate}T00:00:00`);
+
+    if (!isValid(start)) {
+        start = new Date();
+    }
+
+    const today = new Date();
+
+    const monthsDiff = differenceInMonths(today, start);
+
+    let nextBillingDate = addDays(addMonths(start, monthsDiff), 1);
+
+    if (nextBillingDate.toUTCString() < today.toUTCString()) {
+        nextBillingDate = addMonths(nextBillingDate, 1);
```

Comment:
Is this fix to always ensure that the nextBillingDate is in the future? In this case should this be
```suggestion
        nextBillingDate = addMonths(today, 1);
```

 ?

Reply:
nope 

In this case, we will have today's as a billing date